index.html

```diff
@@ -63,133 +70,158 @@ <h4 class="connect">LET'S CONNECT</h4>
     <!--WORKS-->
     <section class="grid-container">
       <div class="cards">
-        <div class="cards-img">
-          <img src="./img/snapshoot-portfolio.png" alt="card-image" />
-        </div>
-        <h2 class="h2">Tonic</h2>
-
-        <ul class="top">
-          <li class="canopy">CANOPY</li>
-          <li><img src="./img/counter.png" alt="ash-counter" /></li>
-          <li class="dev">Back End Dev</li>
-          <li><img src="./img/counter.png" alt="ash-counter" /></li>
-          <li class="dev">2015</li>
-        </ul>
 
-        <p class="para">
-          A daily selection of privately personalized reads; no accounts or
-          sign-ups required.
-        </p>
-
-        <div class="projects">
-          <ul>
-            <li>html</li>
-            <li>css</li>
-            <li>javascript</li>
-          </ul>
-        </div>
+          <div class="cards-img">
+            <img class="tonic-img" src="./img/snapshoot-portfolio.png" alt="card-image" />
+            <img class="desktop-img" src="./img/snapshoot-portfolio-desktop1.png" alt="nature-img">
+          </div>
+          
+          <div class="content-wrap">
+            
+            
+            <h2 class="h2">Tonic</h2>
+            
+            <ul class="top">
+              <li class="canopy">CANOPY</li>
+              <li><img src="./img/counter.png" alt="ash-counter" /></li>
+              <li class="dev">Back End Dev</li>
+              <li><img src="./img/counter.png" alt="ash-counter" /></li>
+              <li class="dev">2015</li>
+            </ul>
+            
+            <p class="para">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            
+            <div class="projects">
+              <ul>
+                <li>html</li>
+                <li>css</li>
+                <li>javascript</li>
+              </ul>
+            </div>
+            
+            <div>
+              <button type="submit" class="btn">See Project</button>
```

Comment:
- [ ] Please use the correct border color for all of your buttons to match the design on the template.

### Your design

![image](https://user-images.githubusercontent.com/83803414/160193265-dec68908-0fe0-4548-bf19-4c2779ab5581.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/160193327-4c0548ff-40f3-4480-995f-6621b83774d0.png)